assignment2.txt

```diff
@@ -0,0 +1,20 @@
+Olivier Gabison - Answers to Part 2 Questions
+
+1. Yes, for example you can have:
+int x = (10 * 20)/2 //This has two operators. You can keep adding operators, there is no "maximum" amount
+of numbers you can't have.
+
+2. An example of a logical error would be misspelling words such as 'int' as 'itn'. Another
```

Comment:
Hm, I think you confused logical errors with syntax ones. You also didn't really compare the two.